plugins/modules/vmware_guest_instant_clone.py

```diff
@@ -84,7 +84,48 @@
       - C(Resources) is the default name of resource pool.
     type: str
     required: False
-
+  guestinfo_vars:
+    description:
+      - Provides GuestOS Customization functionality in instant cloned VM.
+      - A list of key value pairs that will be passed to the destination VM.
+      - These pairs should be used to provide user-defined customization to differentiate the destination VM from the source VM.
+    suboptions:
+      hostname:
+        description:
+          - User specific hostname value which is child of guestinfo.ic option.
+        type: str
+      ipaddress:
+        description:
+          - User specific ipaddress value which is child of guestinfo.ic option.
+        type: str
+      netmask:
+        description:
+          - User specific netmask value which is child of guestinfo.ic option.
+        type: str
+      gateway:
+        description:
+          - User specific gateway value which is child of guestinfo.ic option.
+        type: str
+      dns:
+        description:
+          - User specific dns value which is child of guestinfo.ic option.
+        type: str
+      networktype:
+        description:
+          - User specific networktype value which is child of guestinfo.ic option.
```

Comment:
What can we specify parameters in this option?  
static or dhcp?  
If it is correctly, could you please add the parameter that can be specified to the description?